cnesreport.properties

```diff
@@ -1,14 +1,19 @@
 category=Visualization/Reporting
 description=CNES plugin that allows users to download a bundle of project reports in multiple formats.
 homepageUrl=https://github.com/lequal/sonar-cnes-report
-archivedVersions=
-publicVersions=3.1.0
+archivedVersions=3.1.0
+publicVersions=3.2.2
 
 defaults.mavenGroupId=fr.cnes.sonarqube.plugins
 defaults.mavenArtifactId=cnesreport
 
+3.2.2.description=Support SonarQube 8 up to 8.2 & add ability to export from a specific branch
```

Comment:
See my comment about versions & forward compatibility. Suggest

> Support SonarQube 8.0+

Reply:
See my previous comment on `sqVersions`.

Reply:
Given our other conversations, I'm resolving this one.